packages/compile/package.json

```diff
@@ -3,17 +3,17 @@
   "version": "0.7.9",
   "description": "The core Vensim to C compiler for the SDEverywhere tool suite.",
   "type": "module",
-  "files": [
```

Comment:
This `files` section is effectively replaced by the `.npmignore` file, which is the only reliable way to filter out things like test files that we don't want in the published package.